vim_snippet.vim

```diff
@@ -0,0 +1,25 @@
+function! s:RunBlackMacchiato(visual) range
+    let cmd = "black-macchiato"
+    if !executable(cmd)
+        echo "black-macchiato not found!"
```

Comment:
hmmm, vim does not have an error reporting function?

Reply:
You're right. I will use a different approach.

Reply:
Done